easybuild/easyconfigs/o/Octave/Octave-3.8.2-foss-2015a.eb

```diff
@@ -0,0 +1,26 @@
+name = 'Octave'
+version = '3.8.2'
+
+homepage = 'http://www.gnu.org/software/octave/'
+description = """GNU Octave is a high-level interpreted language, primarily intended for numerical computations."""
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://ftpmirror.gnu.org/octave/']
+
+sanity_check_paths = {
+    'files': ['bin/octave'],
+    'dirs': []
+}
+
+configopts = ['--with-blas="-lopenblas"']
```

Comment:
please use `$LIBBLAS` rather than hardcoding `-lopenblas`, to make this toolchain agnostic

with the `foss` and `goolf` toolchains, `eb` will set `$LIBBLAS` to `-lopenblas -lgfortran`

with the `intel` and `ictce` toolchains, `eb` will set `$LIBBLAS` to something like `-Wl,-Bstatic -Wl,--start-group -lmkl_intel_lp64 -lmkl_sequential -lmkl_core -Wl,--end-group -Wl,-Bdynamic`

this is currently poorly documented, we need to look into that, but the different environment variables set by `eb` in the build environment are listed in a log file, e.g.:

```
== 2015-01-20 15:38:58,193 main.environment INFO Environment variable LIBBLAS set to -lopenblas -lgfortran
```